src/components/Fullscreen.jsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react/prop-types */
```

Comment:
nécessaire ici le disable ?

Reply:
yes totally, Bastien is ok with this one ^^
sharp eyes you have ;)

Reply:
ok then ;)